.github/workflows/check-rtw.yml

```diff
@@ -0,0 +1,83 @@
+# This workflow performs various validation steps for Cylc and Rose.
+name: Check Recipe Test Workflow (RTW)
+
+# Controls when the action will run
+on:
+  # Triggers the workflow on push events
+  push:
+    paths:
+#      - esmvaltool/utils/recipe_test_workflow/**
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
@ehogan I am fairly sure [workflow_dispatch](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch) will trigger everytime there is a push to the `main` branch - once this PR gets merged, that is. You want this behaviour? If not you can set a cron job a la:
```
on:
  # Trigger on push on main or other branch for testing
  # NOTE that push: main will create the file very often
  # and hence lots of automated PRs
  # push:
  #   branches:
  #     - main
  schedule:
    - cron: '0 4 */10 * *'
```
this runs every 10 days at 4am

Reply:
it's actually worse, it triggered at every commit on every branch :grin:

Reply:
Done in #3823 😊